warehouse/postgres/postgres_test.go

```diff
@@ -15,6 +18,104 @@ import (
 	warehouseutils "github.com/rudderlabs/rudder-server/warehouse/utils"
 )
 
+func TestIntegrationPostgresThroughTunnelling(t *testing.T) {
```

Comment:
General question: How do you mock the endpoint to get the secret private key?

Reply:
The part to get the secret key is happening when we fetch the workspace json. Here we provide the workspace json mocked ourselves so no need to mock that function.